o/opentelemetry-js-contrib/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "package_name" : "opentelemetry-js-contrib",
+    "github_url":"https://github.com/open-telemetry/opentelemetry-js-contrib",
+    "version": "auto-instrumentations-node-v0.36.0",
+    "package_dir": "o/opentelemetry-js-contrib",
+    "docker_build": false,
+    "build_script" : "opentelemetry-js-contrib_ubi_8.5.sh",
+    
+    "auto-instrumentations-node-v0.*.*" : {
```

Comment:
Discuss with @seth-priya & @adilhusain-s to confirm on which versions we are going to support.

Reply:
@Pratikt2312 
Please discuss with team and confirm on version, rest is looks good to me.

Reply:
ok sure.